FodyPackaging/build/FodyPackaging.targets

```
@@ -9,19 +9,19 @@
       <NetClassicFilesToInclude Include="$(WeaverDirPath)\net4*\$(PackageId).pdb" />
       <NetStandardFilesToInclude Include="$(WeaverDirPath)\netstandard2*\$(PackageId).dll" />
       <NetStandardFilesToInclude Include="$(WeaverDirPath)\netstandard2*\$(PackageId).pdb" />
+
+      <TfmSpecificPackageFile Include="@(NetClassicFilesToInclude)" PackagePath="netclassicweaver" />
+      <TfmSpecificPackageFile Include="@(NetStandardFilesToInclude)" PackagePath="netstandardweaver" />
+      <TfmSpecificPackageFile Include="$(WeaverPropsTempFile)" PackagePath="build" />
     </ItemGroup>
 
     <Copy SourceFiles="$(WeaverPropsTemplate)" DestinationFiles="$(WeaverPropsTempFile)" />
 
-    <CreateItem Include="@(NetClassicFilesToInclude)" AdditionalMetadata="PackagePath=netclassicweaver">
-      <Output TaskParameter="Include" ItemName="TfmSpecificPackageFile" />
-    </CreateItem>
-    <CreateItem Include="@(NetStandardFilesToInclude)" AdditionalMetadata="PackagePath=netstandardweaver">
-      <Output TaskParameter="Include" ItemName="TfmSpecificPackageFile" />
-    </CreateItem>
-    <CreateItem Include="$(WeaverPropsTempFile)" AdditionalMetadata="PackagePath=build">
-      <Output TaskParameter="Include" ItemName="TfmSpecificPackageFile" />
-    </CreateItem>
+    <PropertyGroup>
+      <AllWeaverFilesIncluded>@(NetClassicFilesToInclude)@(NetStandardFilesToInclude)</AllWeaverFilesIncluded>
+    </PropertyGroup>
+
+    <Error Text="FodyPackaging: No weavers found to include in package. Maybe the build order is wrong?" Condition="'$(AllWeaverFilesIncluded)' == ''" />

```

Wouldn't it be better to raise an error if any one of the weavers is missing? I mean when either netstandard or netclassic is missing.
Having one but not the other could happen if you build projects in parallel.

---

Good point, I'll gonna change this.